sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/tracing/SentryTransactionAdvice.java

```diff
@@ -68,7 +69,7 @@ public Object invoke(final @NotNull MethodInvocation invocation) throws Throwabl
       } else {
         operation = "bean";
       }
-      scopes.pushScope();
+      final @NotNull ISentryLifecycleToken lifecycleToken = scopes.pushIsolationScope();
```

Comment:
Should we move the code below into the try-catch block and use try-with-resource?

Reply:
Done in a follow-up PR